.github/workflows/docker-build.yml

```diff
@@ -31,11 +41,23 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Release name
+        id: release_name
+        uses: actions/github-script@v4
+        with:
+          result-encoding: string
+          script: |
+            const branch_name = process.env.GITHUB_REF
+            const release_version = branch_name.substring(branch_name.lastIndexOf('/') +1 ,branch_name.length  )
```

Comment:
Will need to update this script to also get the version from the tag

Reply:
how do we do that? just using tags and not release branches is also fine I think if that makes it easier

Reply:
We  can remove the branch release then, [da72c7b](https://github.com/celestiaorg/celestia-app/pull/881/commits/da72c7bd8dd39d2cb3219d21e039b34473494195) does them both, but  I guess it's a bit complicated.  What do you think? should I remove getting the version from the branch name?